src/server/routes/calendar.js

```diff
@@ -16,10 +17,10 @@ router.get('/', function(req,res){
 });
 
 router.get('/getevents', function(req, res) {
-
+  //Initiate google_calendar with token
```

Comment:
`// Initiate google_calendar with token`
Always a space after comment block starts before the first word. This is one of those famous things programmers love to get annoyed over.